test/src/Makefile

```diff
@@ -110,6 +110,11 @@ $(ALL_DIRS):
 %/hellobundle.so: $(LIBHELLO_SRC) %
 	$(CC) $(ARCH_FLAGS) -bundle $< -o $@
 
+# Run post-make builds (inconsistent binaries)
+.PHONY: post-make
+post-make:
+	bash post-make.sh $(USE_DIRS)
```

Comment:
How do you feel about making `post-make.sh` executable and then executing it directly via `./post-make.sh` in this line? It feels a bit strange to have a shebang in that file and then to ignore it.